conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalyticsIntegration.java

```diff
@@ -230,95 +176,102 @@ public void reportPlaybackDeficiency(String message, Client.ErrorSeverity severi
      * Puts the session state in a notMonitored state.
      */
     public void pauseTracking() {
-        try {
-            client.adStart(sessionId, Client.AdStream.SEPARATE, Client.AdPlayer.SEPARATE, Client.AdPosition.PREROLL);
-            client.detachPlayer(sessionId);
-            Log.d(TAG, "Tracking paused.");
-        } catch (ConvivaException e) {
-            Log.e(TAG, e.getLocalizedMessage());
+        Log.d(TAG, "Will report ad break started: " + ConvivaSdkConstants.AdPlayer.SEPARATE + " " + ConvivaSdkConstants.AdType.CLIENT_SIDE);
+        convivaVideoAnalytics.reportAdBreakStarted(ConvivaSdkConstants.AdPlayer.SEPARATE, ConvivaSdkConstants.AdType.CLIENT_SIDE);
+        if(isSessionActive) {
+            convivaVideoAnalytics.reportPlaybackEnded();
+            isSessionActive = false;
         }
 
+        Log.d(TAG, "Tracking paused.");
     }
 
     /**
      * Puts the session state from a notMonitored state into the last one tracked.
      */
     public void resumeTracking() {
-        try {
-            client.attachPlayer(sessionId, playerStateManager);
-            client.adEnd(sessionId);
-            Log.d(TAG, "Tracking resumed.");
-        } catch (ConvivaException e) {
-            Log.e(TAG, e.getLocalizedMessage());
+        if(!isSessionActive) {
+            convivaVideoAnalytics.reportPlaybackRequested();
```

Comment:
Same as comment for pauseTracking(), does `reportPlaybackRequested` not start a new session instead of restarting monitoring of existing session?

Reply:
Well, we don't actually know how sessions are now handled, behind the scenes. The bitmovin term for session is no longer a match for the Conviva concept. I'm trying to get more info about this than the docs offer (from our Conviva contacts), but this what the discussions above in this PR were about. In this case, we are simply ensuring that the reportPlaybackRequested/reportPlaybackEnded calls are "balanced". Without this, touchstone reports errors basically.

Reply:
thanks @sinewave440hz . I understand your point about sessions but my concern here is more about parity between stopping and resuming monitoring in a session(old code) VS terminating and starting a new session(new code). Is it possible to run a test using pauseTracking() and resumeTracking() APIs with old code and compare the touchstone session report with another session run using new code to check if the outcome looks similar?